src/api.py

```
@@ -825,14 +1102,15 @@ async def websocket_evaluation_progress(websocket: WebSocket, run_id: int):
     
     try:
         while True:
-            # Get current progress
+            # Get current progress using database module directly
+            from .database import async_session
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)